docfx_yaml/extension.py

```diff
@@ -820,17 +832,21 @@ def _update_friendly_package_name(path):
     # Add extracted summary
     if lines != []:
         # Resolve references for xrefs in two different formats.
-        # REF_PATTERN checks for patterns like ":class:`~google.package.module`"
-        lines, xrefs = _resolve_reference_in_module_summary(REF_PATTERN, lines)
-        for xref in xrefs:
-            if xref not in app.env.docfx_xrefs:
-                app.env.docfx_xrefs[xref] = ''
-
-        # REF_PATTERN_LAST checks for patterns like "~package.module"
-        lines, xrefs = _resolve_reference_in_module_summary(REF_PATTERN_LAST, lines)
-        for xref in xrefs:
-            if xref not in app.env.docfx_xrefs:
-                app.env.docfx_xrefs[xref] = ''
+        ref_patterns = [
```

Comment:
Declare this next to the declarations of these vars?

Reply:
Done.